docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -0,0 +1,168 @@
+---
+id: gcp-export-gcs
+title: Exporting Workflow Event History to GCS
+sidebar_label: GCP Export to GCS
+description: Export Workflow History to GCS
+slug: /cloud/export/gcp-export-gcs
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - gcp
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - gcp
+---
+
+## Prerequisites {#prerequisites}
+
+Before configuring the Export sink, please complete the following steps in Google Cloud.
+
+1. Create a GCS bucket and take note of its bucket name, for example, "test-export"
+Enable customer-managed encryption keys (CMEK) if you need additional security for your GCS bucket.
+2. Record the GCP Project ID that owns the bucket.
+Create a service account in the same project that grants Temporal permission to write to your GCS bucket.
+4. Follow the instructions in the Temporal Cloud UI, there are two ways to set up this service account:
+    - Manual Setup:
+      - Input the service account ID, GCP project ID and GCS bucket name.
+      - Follow the instructions, manually set up a new service account.
+    - Automated Setup:
+      - Use the [Terraform template](https://github.com/temporalio/terraform-modules/tree/main/modules/gcp-sink-sa) to create the service account.
+
+## Configure Workflow History Export 
+
+How to configure Workflow History Export using the Temporal Cloud UI or tcld
+
+You can use either the [Temporal Cloud UI](#using-temporal-cloud-ui) or [tcld](#using-tcld) to configure the Workflow History Export.
+
+### Using Temporal Cloud UI
+
+The following steps guide you through setting up Workflow History Export using the Temporal Cloud UI.
+
+![](/img/export-sink-ui-gcp.png)
+
+1. In the Cloud UI, navigate to the Namespaces section. Confirm that the Export feature is visible and properly displayed.
+2. Configure the Export sink for a Namespace:
+    1. Choose GCS as the Sink type.
+    2. Provide the following information:
+        1. Name: [User-defined]
+        2. Service account ID: [from Prerequisite 3]
+        3. GCP Project ID: [from Prerequisite 2]
+        4. GCS bucket name: [from Prerequisite 1]
+3. After inputting the necessary values, click on **Verify**. 
+  You should be able to write to sink successfully. 
+  If not, please fix any errors or reach out to support for help.
+4. Clicking **Create** will complete the Export sink set up
+5. The page will auto-refresh and you should see the status “Enabled” on the Export screen. 
+  You are now ready to export workflow histories.
+6. You can toggle the enable button if you want to stop export and resume in the future. 
+  Note: when you re-enable the feature, it will start from the current point in time, and not from the time when you disable export.
```

Comment:
```suggestion
  Note: when you re-enable the feature, it will start from the current point in time, and not from the time when you disabled export.
```